packages/blockchain-wallet-v4-frontend/src/scenes/Register/template.js

```diff
@@ -155,6 +156,7 @@ const Register = ({
                 data-e2e='signupPassword'
                 name='password'
                 validate={[required, validStrongPassword]}
+                noLastPass
```

Comment:
What if the user uses lastpass to generate a password?

Reply:
I was trying to find a way to detect whether or not users has lp as an extension, but it looks like I wasn't able to find a way so i will remove noLastPass for now